src/components/AdminPage/Layout.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import TopMenu from './TopMenu';
+import Articles from './Articles';
+import SideNavBar from './SideNavBar';
+
+function Layout() {
```

Comment:
It's good that you divided it up 👌 Would a more specific name like `AdminPage` or `AdminLayout` be more appropriate though?

Reply:
AdminLayout would be better you're right Layout is not specific enough